apps/metrics/__init__.py

```diff
@@ -0,0 +1,115 @@
+from common.utils import camel2snake
+
+
+class MetricType:
+    UNKNOWN = 'unknown'
+    TIME = 'time'
+    SIZE = 'size'
```

Comment:
Bandwith and content size is a bit too specific. Size in bytes seems to me a bit too long and descriptive. Any idea or preference?